readme/api/references/plugin_manifest.md

```diff
@@ -75,6 +82,7 @@ If no promo tile is provided, your plugin icon will be displayed instead.
     "version": "1.0.0",
     "author": "John Smith",
     "app_min_version": "1.4",
+    "platforms": ["mobile>=3.0.3", "desktop"],
```

Comment:
An alternative to including minimum versions directly within the platform string could be to make this a dictionary instead. For example,

```suggestion
    "platforms": {"mobile": ">=3.0.3", "desktop": true},
```

The above may be more difficult to work with, however, particularly if most plugins will specify `"platforms": ["desktop"]`.

How other apps handle this:
- Obsidian lets plugins specify a [`isDesktopOnly`](https://docs.obsidian.md/Reference/Manifest#Plugin-specific+properties) property. Joplin will likely need the ability to specify different minimum versions for different platforms, however (reasoning: 1) different mobile and desktop versions correspond to the same commit and 2) plugin APIs may be added to different platforms at different times).
- [Expo uses a list of supported platforms](https://docs.expo.dev/versions/latest/config/app/#platforms), for example `[ "android", "ios" ]`.

Reply:
An alternative could be to add an `app_min_version_mobile` property.